home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/MailHogUtil/Dto/ResponseDto.java

```diff
@@ -0,0 +1,16 @@
+package com.softserveinc.ita.homeproject.api.tests.utils.MailHogUtil.Dto;
+
+import com.softserveinc.ita.homeproject.api.tests.utils.MailHogUtil.ApiMailHogUtil;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+public class ResponseDto {
```

Comment:
As well as having package identity it would be great to add some context to class name so these classes would be much easier to identify and relate to mock emails functionality.

Reply:
Added context to class names